### ISSUE TITLE
Enhance a11y for `Animation`

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#1263]
path: stories/components/motion/animation.stories.tsx

### Description
2 accessibility violations were found in the `Animation`.

#### [withLayout](https://yamada-ui.github.io/yamada-ui/?path=/story/components-motion-animation--with-layout)

Expected the HTML found at $('.css-i2tfac') to have no violations:
    <button class="css-i2tfac"><div class="ui-motion css-xj6vze"></div></button>

    Received:
    Buttons must have discernible text (button-name)

    Fix any of the following:
      Element does not have inner text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element's default semantics were not overridden with role="none" or role="presentation"

    You can find more information on this issue here:
    https://dequeuniversity.com/rules/axe/4.9/button-name?application=axeAPI

#### [useLayoutGroup](https://yamada-ui.github.io/yamada-ui/?path=/story/components-motion-animation--use-layout-group)

Expected the HTML found at $('button[data-projection-id="1"]') to have no violations:
    <button class="ui-motion css-1dxhdff" data-projection-id="1"></button>

    Received:
    Buttons must have discernible text (button-name)

    Fix any of the following:
      Element does not have inner text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element's default semantics were not overridden with role="none" or role="presentation"

    You can find more information on this issue here:
    https://dequeuniversity.com/rules/axe/4.9/button-name?application=axeAPI

Expected the HTML found at $('button[data-projection-id="2"]') to have no violations:
    <button class="ui-motion css-1dxhdff" data-projection-id="2"></button>

    Received:
    Buttons must have discernible text (button-name)

    Fix any of the following:
      Element does not have inner text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element's default semantics were not overridden with role="none" or role="presentation"

    You can find more information on this issue here:
    https://dequeuniversity.com/rules/axe/4.9/button-name?application=axeAPI

Expected the HTML found at $('button[data-projection-id="3"]') to have no violations:
    <button class="ui-motion css-1dxhdff" data-projection-id="3"></button>

    Received:
    Buttons must have discernible text (button-name)

    Fix any of the following:
      Element does not have inner text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element's default semantics were not overridden with role="none" or role="presentation"

    You can find more information on this issue here:
    https://dequeuniversity.com/rules/axe/4.9/button-name?application=axeAPI

Expected the HTML found at $('button[data-projection-id="4"]') to have no violations:
    <button class="ui-motion css-1dxhdff" data-projection-id="4"></button>

    Received:
    Buttons must have discernible text (button-name)

    Fix any of the following:
      Element does not have inner text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
      Element's default semantics were not overridden with role="none" or role="presentation"

    You can find more information on this issue here:
    https://dequeuniversity.com/rules/axe/4.9/button-name?application=axeAPI

### Problem Statement/Justification
A11y is essential to ensure quality when operating a package.

### Additional Information
While it's ideal to cover all jobs, if there are too many jobs to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.
